torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -298,29 +259,30 @@ def consolidate_state_dict(self, to: int = 0) -> None:
                     )
                 else:
                     # Receive the optimizer state from the source rank
-                    local_state_dict = _broadcast_object(
-                        empty_messenger,
+                    object_list = [empty_messenger]
```

Comment:
Minor, but I do not think you need the `empty_messenger` anymore. You can set `object_list = [None]` and get rid of `empty_messenger`.

Reply:
Great point, revised as suggeted